app/services/api.js

```
@@ -101,18 +104,21 @@ function showError(error, status) {
       message = error.result.errorMessage.errors[0].message;
   }
+
   message = message || 'An error occurred with the API';
 
   capNotificationSeverityType({
     message,
   });
 }
+
 function checkStatusCode(res) {
   if (res && res.code) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)